backend/repository/bets.go

```diff
@@ -0,0 +1,48 @@
+package repository
+
+type BetsRepository struct {
+	db Database
+}
+
+func NewBetsRepository(db Database) *BetsRepository {
+	return &BetsRepository{db: db}
+}
+
+func (repo *BetsRepository) FirstTimeBets() (int64, error) {
+	// Define a struct to hold the query result
+	var bets []struct {
+		MarketID string
+		Username string
+	}
+
+	// Execute raw SQL query to group by both market_id and username
+	// gorm GROUP does not allow two inputs, so this has to be a raw function
+	result := repo.db.Raw(`
+		SELECT market_id, username 
```

Comment:
If we did `SELECT DISTINCT ....` would that return the desired result?

Reply:
I don't believe it does, I remember trying that, would have to go back and check this. I'm trying to get a COUNT value which we can then, "Scan" (using Gorm terminology) into an int64.